Makefile

```diff
@@ -92,6 +92,6 @@ endif
 ## HELP ##
 ##########
 help:
-	@grep -h -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+	@grep --extended-regexp '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
Looks like you skipped --no-filename

Reply:
You're right

Reply:
I've checked and this parameter has no effect in our use case

Reply:
It's useful only when we have multiple files. Here we always have only one.

Reply:
Do you recall if you added it for a specific reason @marienfressinaud ? (Like some future usage.)

Reply:
It looks like a copy-&-paste from an internet resource. I've done the same thing at work :)

Reply:
ha ha, yes it is a copy-paste from what I used at work ^^ I think it was initially from [a Marmelab article](https://marmelab.com/blog/2016/02/29/auto-documented-makefile.html). I added the `-h` to avoid problems when including `.env` files in another project though (see in the article comments). I think it is safe to drop it here since we don't include any files.

Reply:
That article led me to this:

https://github.com/dergachev/screengif/blob/a26c9880a09fa4c782abdf61ca496270b055d8e8/Makefile#L20-L21

```
changelog:
		git log `git describe --tags --abbrev=0`..HEAD --oneline |  awk 'BEGIN { print "Commits since last tag:" }; {print "- " $$0}' | vim - -R +"vs CHANGELOG.md" +"set noro"
```

Not really sure I quite see the advantage of all that, but it makes me wonder if a `make log` for `git log --oneline` would be useful. :-)

Reply:
@Frenzie There is a better tool to make that and it's called git aliases.
You can add an alias this way:
```
git config --global alias.oneline 'log --oneline --abbrev-commit --graph'
```
And use it this way:
```
git oneline
```
That's what I use everyday :)

Reply:
Maybe. I'd be more inclined say there's a better tool to do that and it's called something like gitk, QGit or git-dag. :-P

But I just realized they mean "changelog" as a tool to help prepare a manual changelog. :man_facepalming: